NameSorterAlpha/NameSorterAlpha/AlternateName.cs

```diff
@@ -27,24 +27,23 @@ public Person(string info, int line_num)
                 GivenNames += infoArray[i];
                 GivenNames += ' ';
             }
-            GivenNames.TrimEnd(new char[] {' '}); 
+            GivenNames.TrimEnd(new char[] {' '});
 
-            try
-            {
-                _dateOfBirth = DateTime.ParseExact(infoArray[length - 1], dateFormat, CultureInfo.InvariantCulture);
-            }
-            catch (Exception)
+            if (DateTime.TryParseExact(infoArray[length - 1], dateFormat, CultureInfo.InvariantCulture, DateTimeStyles.None, 
```

Comment:
Probably can invert the if clause, so you don't get `if` path and filled `else` path.